LICENSE-BSD-3-Clause

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2018, Tony Atkins
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+
+* Neither the name of the gpii-grunt-lint-all nor the names of its
```

Comment:
It seems hard to imagine this name being used to endorse or promote anything :)

Reply:
You caught me, that bit of boilerplate game from `grunt init`.  Removed in favour of a modified version of the license in universal (updated to 2018, minus the funding wording).